sdk/proto/opentelemetry/README

```diff
@@ -0,0 +1,2 @@
+From: https://github.com/open-telemetry/opentelemetry-proto
```

Comment:
bazel pulls this in as an external repository, but we also vendor it?

Reply:
That's for cmake. 

I'm open to other ways of supporting it, but I've usually found this to be the path of least pain for cmake.

Reply:
What do you think about `FetchContent` in CMake?

Reply:
Can you do offline builds with FetchContent?

This was something asked for with jaeger's project that used similar fetching.
https://github.com/jaegertracing/jaeger-client-cpp/issues/38

Reply:
I think the short answer is no. :(

When this came up in opencensus-cpp, what we wanted is to have a choice between:
 * automatically download dependencies as part of the build because that makes it easy to get started
 * or: opt out of the download and use a locally installed dependency

If you look at https://github.com/census-instrumentation/opencensus-cpp/blob/2d0ed08e90c15388f9803c91ef61de2c179ead8c/cmake/OpenCensusDeps.cmake it does "if(NOT abseil_POPULATED) FetchContent_Populate(abseil)"

The intent was: if you're using CMake to build your app, you can pull in opencensus-cpp and other deps in your outermost CMakeFiles.txt, similar to how bazel "solves" this problem by inflicting it on the outermost WORKSPACE.

I honestly don't know what the right thing is here. :(

I'm not thrilled about vendoring. I'm slightly less thrilled about both vendoring and downloading depending on the build system.

Reply:
I set both bazel and cmake to point to the same vendored version. And the protobuf package is still exposed in the WORKSPACE so that projects can still specify the dependency.

Reply:
Thanks. At least this way it's consistent. :)

I don't know what the long-term answer is for CMake but that's out of scope for this PR.